### PR TITLE
Feature/buttons

### PR DESCRIPTION
**Pull request by danielmurray**
* Cleaned up Icons + Buttons
 * Better Ripples
 * Added tooltip
* Added material icons support
* Added basic name-spacing for updated views

### CODE REVIEW

File: src/js/appbar.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+(function (Wood) {
+    Wood.AppBar = Marionette.LayoutView.extend({
+      tagName: "appbar",
+      attributes: {
+        class: 'appbar',
+      },
+      template: _.template(''+
+        '<div class="appbar-brand" href="#">' +
+          '<div id="avatar-wrapper" class="avatar-wrapper"></div>' +
+          '<div class="tesla-logo"></div>' +
```
Comment (erichulser) [inline src/js/appbar.js]: gotta be careful here...

File: src/sass/_appbar.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.appbar{
```
Comment (erichulser) [inline src/sass/_appbar.scss]: if you're worried about how bootstrap is global, we should avoid doing it here too.  If you nest everything under a `.wood` class, then you can allow people to be more target width their integration:

    .wood {
        .appbar {
            ...
         }
    }

this way, if you're using wood for your whole site, you can just do `<body class="wood">` but if not, you can be more specific.

File: src/js/table.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
+(function (Wood) {
+    Wood.Table = Marionette.ItemView.extend({
```
Comment (erichulser) [inline src/js/table.js]: we should talk a bit how to make this more generic -- this is going to be pretty specific to ORB responses right now I think

File: src/sass/_appbar.scss (hunk 2 above)
Comment (danielmurray) [inline src/sass/_appbar.scss]: Right, I was considering that.. I was thinking of integrating it into the sass

File: src/js/table.js (hunk 3 above)
Comment (danielmurray) [inline src/js/table.js]: I haven't touched the Table code, that is a TODO, however, my plan is to just have a base table class that takes a collection as an argument

Comment (erichulser) [inline src/js/table.js]: I'm not sure we want the table here actually, at least not the DataTable one.  That'll be more specific to ORB and orb.js

**Pull request merged**